src/QirRuntime/test/QIR-static/CMakeLists.txt

```diff
@@ -20,7 +20,8 @@ add_custom_target(qir_static_test_lib DEPENDS ${QIR_TESTS_LIBS})
 # The executable target for QIR tests triggers the custom actions to compile ll files
 #
 add_executable(qir-static-tests
-  qir-driver.cpp)
+  qir-driver.cpp
+  qir-test-math.cpp)

```

Comment:
Nit: maybe, move the closing parenthesis to a new line, so future additions look cleaner in diffs?

Reply:
Fixed.